openmdao/test_suite/components/matmultcomp.py

```diff
@@ -45,11 +45,6 @@ def compute(self, inputs, outputs):
         time.sleep(self.sleep_time)
 
```

Comment:
Do we even need the MatMultComp anymore?

Reply:
Yeah it's still used for parallel FD testing.